proofpoint_tap/plugin.spec.yaml

```diff
@@ -4,7 +4,7 @@ products: [insightconnect]
 name: proofpoint_tap
 title: Proofpoint TAP
 description: A plugin for Proofpoint Targeted Attack Protection (TAP) alerts
-version: 1.0.4
+version: 1.0.5
```

Comment:
This should be a minor version bump per https://semver.org which is what we use.